data_prep/project_context/context_retriever.py

```diff
@@ -240,8 +240,10 @@ def _get_namespace_node(self, namespace_node, current_depth: int) -> bool:
         if node.get('name') == self._function_name:
           return True
 
-      if node.get('kind') == 'NamespaceDecl' and current_depth < len(self._namespace_tokens) and self._get_namespace_node(node, current_depth):
-          return True
+      if node.get('kind') == 'NamespaceDecl' and current_depth < len(
+          self._namespace_tokens) and self._get_namespace_node(
+              node, current_depth):
```

Comment:
Add `()` around conditions because we cannot fit them in one line. For example:
https://github.com/google/oss-fuzz-gen/pull/86#discussion_r1483929208